sources/packages/api/src/esdc-integration/e-cert-full-time-integration/e-cert-full-time-integration.service.ts

```diff
@@ -0,0 +1,206 @@
+import { Injectable } from "@nestjs/common";
+import {
+  ConfigService,
+  SequenceControlService,
+  SshService,
+} from "../../services";
+import { ESDCIntegrationConfig } from "../../types";
+import {
+  getDayOfTheYear,
+  getDisbursementValuesByType,
+  getGenderCode,
+  getMaritalStatusCode,
+  getTotalDisbursementAmount,
+  getTotalYearsOfStudy,
+} from "../../utilities";
+import {
+  CreateRequestFileNameResult,
+  ECertRecord,
+  NUMBER_FILLER,
+  RecordTypeCodes,
+} from "./models/e-cert-full-time-integration.model";
+import { StringBuilder } from "../../utilities/string-builder";
+import { EntityManager } from "typeorm";
+import { SFTPIntegrationBase } from "../../services/ssh/sftp-integration-base";
+import { FixedFormatFileLine } from "../../services/ssh/sftp-integration-base.models";
+import { ECertFileHeader } from "./e-cert-files/e-cert-file-header";
+import { ECertFileFooter } from "./e-cert-files/e-cert-file-footer";
+import { ECertFileRecord } from "./e-cert-files/e-cert-file-record";
+import { DisbursementValueType } from "../../database/entities";
+
+/**
+ * Manages the file content generation and methods to
+ * upload/download the files to SFTP.
+ */
+@Injectable()
+export class ECertFullTimeIntegrationService extends SFTPIntegrationBase<void> {
+  private readonly esdcConfig: ESDCIntegrationConfig;
+
+  constructor(
+    config: ConfigService,
+    sshService: SshService,
+    private readonly sequenceService: SequenceControlService,
+  ) {
+    super(config.getConfig().zoneBSFTP, sshService);
+    this.esdcConfig = config.getConfig().ESDCIntegration;
+  }
+
+  /**
+   * Create the ECert file content, by populating the
+   * header, detail and trailer records.
+   * @param ecertRecords student, User and application data.
+   * @param fileSequence unique file sequence.
+   * @param totalSINHash sum hash total of the Student's SIN.
```

Comment:
Can we please remove this totalSINHash param in the comments?

Reply:
Removed!